fetch2/src/main/java/com/tonyodev/fetch2/HttpUrlConnectionDownloader.kt

```diff
@@ -193,6 +193,16 @@ open class HttpUrlConnectionDownloader @JvmOverloads constructor(
     }
 
     override fun getRequestSupportedFileDownloaderTypes(request: Downloader.ServerRequest): Set<Downloader.FileDownloaderType> {
+        return when (fileDownloaderType) {
+            Downloader.FileDownloaderType.SEQUENTIAL -> mutableSetOf(fileDownloaderType)
+            else -> try {
+                getRequestSupportedFileDownloaderTypes(request, this)
+            } catch (e: Exception) {
+                mutableSetOf(fileDownloaderType)
+            }
+            return mutableSetOf(fileDownloaderType)
```

Comment:
probably not needed anymore

Reply:
True (sorry I missed that - still waking up)... I will amend the pull request later this afternoon to reflect that.